bbp/modules/default.nix

```diff
@@ -2467,6 +2467,7 @@ let
         		pydot
         		pillow
         		ipyparallel
+                jinja2
```

Comment:
Do I get it right that the module I'd like to deploy should appear in 'all' somehow?
Does it belong here?

Reply:
Yes. Or in the modules.set subgroups.

"all" is the target which is triggered for dereployment and also for testing